runtime/src/consensus/tendermint/verifier.rs

```diff
@@ -529,6 +533,23 @@ impl Verifier {
         Ok(state)
     }
 
+    fn update_insecure_posix_time(&self, verified_block: &TMLightBlock) {
+        // Update untrusted time if ahead. This makes sure that the enclave's sense of time is
+        // synced with consensus sense of time based on the fact that consensus time is harder to
+        // fake than host operating system time.
```

Comment:
prior to this, were we using host operating system time? this commit doesn't remove any other calls to time::update_insecure_posix_time. and the API looks like it allows any caller to overwrite any previously set time. so will this harder-to-fake consensus time unpredictably battle easier-to-fake system time?

Reply:
Previously (and still do with IAS quotes) we were also using IAS time which did battle with system time :) That is why enclaves aborted on some nodes with desynced clocks.